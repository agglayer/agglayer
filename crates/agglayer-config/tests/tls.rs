use agglayer_config::TlsConfig;

#[test]
fn tls_config_serialization() {
    let config = TlsConfig {
        certificate: "/path/to/cert.pem".into(),
        key: "/path/to/key.pem".into(),
    };

    insta::assert_toml_snapshot!(config);
}
