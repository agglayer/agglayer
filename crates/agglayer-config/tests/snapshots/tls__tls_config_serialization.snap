---
source: crates/agglayer-config/tests/tls.rs
expression: config
snapshot_kind: text
---
certificate = "/path/to/cert.pem"
key = "/path/to/key.pem"
